plugins/acss-kit/assets/utilities/display.css

```
@@ -0,0 +1,41 @@
+/* display utilities */
+.hide { display: none !important; }
+.show { display: revert !important; }
+.invisible { visibility: hidden !important; }
+.sr-only {
+  position: absolute; width: 1px; height: 1px;
+  padding: 0; margin: -1px; overflow: hidden;
+  clip: rect(0, 0, 0, 0); white-space: nowrap; border-width: 0;
```

Stylelint (17.11.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)